api/v1/models/User.js

```diff
@@ -56,11 +60,10 @@ User.create = function (email, password, role) {
 	if(!role){
 		// No roles were provided, so create the User
 		return user.setPassword(password)
-		.then(function(result){
-			return result.save();
-		});
+			.then(function(result){
+				return result.save();
+			});
 	}
-
```

Comment:
Probably want this newline back since it separates the following transaction into its own block